resources/user.rb

```diff
@@ -0,0 +1,106 @@
+property :username, String, name_attribute: true, desired_state: false, identity: true
+property :password, String, desired_state: false, identity: true, sensitive: true
+property :first_name, String, desired_state: false, identity: true, default: ''
+property :last_name, String, desired_state: false, identity: true, default: ''
+property :email, String, desired_state: false, identity: true, default: ''
+property :roles, Array,  desired_state: false, identity: true, default: []
```

Comment:
Above properties are `desired_state`, which is the default. Please remove `desired_state: false, identity: true`

Reply:
done